addons/website/static/src/js/content/menu.js

```diff
@@ -640,11 +644,8 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
         if (config.device.size_class <= config.device.SIZES.SM) {
             return;
         }
-
-        const $dropdown = $(ev.currentTarget);
-        $dropdown.addClass('show');
-        $dropdown.find(this.$dropdownToggles).attr('aria-expanded', 'true');
```

Comment:
Is it alright to not have the `aria-expanded` attribute anymore ? It works but I am just wondering what this attribute was doing and why it was needed ? :thinking:

Reply:
Okay, then I will consider it as a manual addition that should not have been made this way (like the `show` classes)
It doesn't seem to cause any problem to remove it